### ISSUE TITLE
Cannot archive completed todos

### ISSUE THREAD

**Issue by slehan** [ultralist/ultralist#185]
Before todolist was changed to ultralist, you could do `todolist archive completed` or something like that to archive all todos you had in the completed state. [This seems to be possible in ultralist](https://docs.ultralist.io/cli.html#archiving_completed) but the example doesn't work:

```shell
~> ultralist a completed
Todo 66 added.
```

This just adds a todo that says "completed". I also tried the following but to no avail:
```shell
~> ultralist ar completed
Invalid id.
Invalid id: ar completed.
~> ultralist archive completed
Invalid id.
Invalid id: archive completed.
```

I loved this feature before, maybe I'm doing it wrong? However, it seems like its a bug.

**Comment by slehan** [comment]
Seems to have been fixed in 0.9.3, updating the Homebrew formulae to use 0.9.3 now.

**Comment by gammons** [comment]
hi @slehan , yeah the syntax changed slightly.  it should work in 0.9.3 and up.  closing for now, but please re-open if this is not working for you.  Also 1.0 will be coming out very shortly.
